src/unitxt/standard.py

```diff
@@ -29,7 +34,7 @@ class BaseRecipe(Recipe, SourceSequentialOperator):
     card: TaskCard
     template: Template = None
     instruction: Instruction = None
-    format: ICLFormat = ICLFormat()
+    format: SystemFormat = SystemFormat()
 
```

Comment:
@elronbandel , I add the comment here but this refers to `verify` method, that I have not changed, hence Github would not allow to add comments to. a couple of lines down.
It seems that (in some cases) `ValueError` is raised DAVKA when the desired condition, as described in the exception text, **is** satisfied. Shall I swap the condition, or the text?. In most cases, the text seems to me correct (up to the pettiness with the == case), but I wanted to check with you.